### DISCUSSION TITLE
Re possible merging of Plus wih Plus-Plus

### DISCUSSION THREAD

**Discussion by fade2gray** [mark-wiemer/ahkpp#23] [General]
Please, would you consider **_not_** including the keyboard shortcut for accessing AHK's documentation introduced [here](https://github.com/AutoHotkey-Plus/vscode-autohotkey/issues/55)?

I could be wrong but I believe F1 has always been a VSC default keybinding along with Ctrl+Shift+P for accessing the command palette. Personally, I simply access the documentation from a shortcut placed in Window's quick-launch toolbar and trying to un-train muscle-memory for using F1 to access the command palette has proved to be a little irritating.

**Comment by FuPeiJiang** [comment]
thanks, I never knew F1 opened command palette, that's way easier.

you don't want F1 to be remapped to `"command": "document.open",` (it wasn't really clear)

I just tested the functionality of `"command": "document.open",`, all it does is open up the document..., it doesn't even go to the selected ahk command. I wouldn't want that too

Personally, if it's to search an ahk command, I double click the command (for example WinGetTitle) to select it and I use ctrl+right click
```autohotkey
$^Rbutton::
    gotClip:=getClip()
    runCommand := """C:\Program Files (x86)\Google\Chrome\Application\chrome.exe"" ""https://www.autohotkey.com/docs/commands/" gotClip ".htm"""
    Run, %runCommand%
return
getClip()
{
    clipboard=
    Send, ^c
    Send, ^c
    sleep, 50
return clipboard
}
```

**Comment by mark-wiemer** [comment]
Yeah, that seems like a major oversight. I will do my best not to use keybindings that are also defaults in VS Code. `Shift + F1` seems like a fine replacement. Thanks for bringing this up
